test/extended/networking/util.go

```diff
@@ -160,12 +161,10 @@ func pluginIsolatesNamespaces() bool {
 }
 
 func makeNamespaceGlobal(ns *api.Namespace) {
-	client, err := testutil.GetClusterAdminClient(testexutil.KubeConfigPath())
+	client, err := testutil.GetClusterAdminKubeClient(testexutil.KubeConfigPath())
```

Comment:
oh, this is actually much easier now that you only need a kubeclient. you can have the caller pass the whole *e2e.Framework object rather than just f.Namespace, and then use f.Client